tests/src/main/scala/sttp/tapir/tests/package.scala

```diff
@@ -97,6 +97,8 @@ package object tests {
   val in_input_stream_out_input_stream: Endpoint[InputStream, Unit, InputStream, Any] =
     endpoint.post.in("api" / "echo").in(inputStreamBody).out(inputStreamBody).name("echo input stream")
 
+  val in_empty_out_empty: Endpoint[Unit, Unit, Unit, Any] = endpoint.post.in("api" / "empty").in(emptyInput).out(emptyOutput)
```

Comment:
isn't this just `endpoint.post.in("api" / "empty")`? Also, maybe we can simply use e.g. `in_root_path`